web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -48,6 +48,7 @@ const authSlice = createSlice({
       state.authenticating = false
       state.isAuthenticated = false
       state.error = action.payload
+      state.roles = []
```

Comment:
Can we test this file? It doesn't get pulled in as part of cypress, but maybe we can do a normal boring unit test on it? (Either that - or we should just throw it into codecov ignore).
Authentication and role extraction seems pretty import though?

Reply:
Done in: https://github.com/bcgov/wps/pull/1883/commits/78ae303d7fa1721bc40c078f86835a4f5f2626dd